### PR TITLE
In cat app photo project, Mislabeled meta element changed to tag

### PR DESCRIPTION
**Pull request by Kshitiz-Mhto**
#46772

Checklist:
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ dashedName: step-68
 
 # --description--
 
-One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` tag as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
+One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `<meta>` tag as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:10]: I think, if I'm understanding correctly, that we have a meta element and not a meta tag? In any case, you have a mix of both terminology in this PR, so something will need to be updated

Comment (jeremylt) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:10]:
```suggestion
One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` elebent as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md [hunk 2]
```diff
@@ -7,17 +7,17 @@ dashedName: step-68
...
 
 # --hints--
 
-You should create a `meta` element within the `head` element.
+You should create a `<meta>` element within the `head` element.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:14]:
```suggestion
You should create a `meta` element within the `head` element.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md [hunk 3]
```diff
@@ -7,17 +7,17 @@ dashedName: step-68
...
 
 ```js
 assert.exists(document.querySelector('head > meta'));
 ```
 
-You should give the `meta` tag a `charset` of `UTF-8`.
+You should give the `<meta>` tag a `charset` of `UTF-8`.
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md:20]:
```suggestion
You should give the `meta` element a `charset` of `UTF-8`.
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
@Kshitiz-Mhto, I agree with jeremylt. 

The steps, that have tests checking for opening and closing tags `<>`, are the ones we use tags with.

Here is an example:

> Your `img` element should not have a closing tag. Closing tags have a `/` just after the `<` character.
>
> ```js
> assert(!code.match(/<\/img\>/));
> ```

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Kshitiz-Mhto, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by Sembauke — APPROVED** [review]

**Pull request merged**